crates/cli-support/src/js/mod.rs

```diff
@@ -644,6 +648,59 @@ impl<'a> Context<'a> {
             ));
         }
 
+        // If the class is inspectable, generate `toJSON` and `toString`
+        // to expose all readable properties of the class. Otherwise,
+        // the class shows only the "ptr" property when logged or serialized
+        if class.is_inspectable {
+            // Creates a JavaScript object of all readable properties
+            // This looks like { a: this.a, b: this.b }
+            let readable_properties_js_object = format!(
+                "{{{}}}",
+                class
+                    .readable_properties
+                    .iter()
+                    .fold(String::from("\n"), |fields, field_name| {
+                        format!("{}{name}: this.{name},\n", fields, name = field_name)
+                    })
+            );
+
+            dst.push_str(&format!(
+                "
+                toJSON() {{
+                    return {readable_properties};
+                }}
+
+                toString() {{
+                    return JSON.stringify({readable_properties});
+                }}
+                ",
+                readable_properties = readable_properties_js_object
+            ));
+
+            if self.config.mode.nodejs() {
+                // `util.inspect` must be imported in Node.js to define [inspect.custom]
+                self.import_name(&JsImport {
+                    name: JsImportName::Module {
+                        module: "util".to_string(),
+                        name: "inspect".to_string(),
+                    },
+                    fields: Vec::new(),
+                })?;
```

Comment:
The return value of this is a `String` which I think should be used instead of `import.custom` below

Reply:
I'm not sure if I can, this import gives me "inspect" back but the symbol name has to be "inspect.custom"… Do you know if there's another method I should try to get the symbol name in a variable?

Reply:
Oh sorry what I mean is that below instead of `[inspect.custom]` you'll want to do `[{}.custom]` where `{}` is the return value of this function. That way if the identifier `inspect` is imported from elsewhere it'll still work.

Reply:
That makes sense! I've updated it